### PR TITLE
DRYs up formatting to a single task, fixes quotes issue

### PR DESCRIPTION
**Pull request by mikepw**

### CODE REVIEW

File: ansible/roles/terraform/tasks/main.yml [hunk 1]
```diff
@@ -10,27 +10,19 @@
...
+    msg: "{{kv_tf | to_nice_json}}"
 
 
 - name: tf plan
   terraform:
     project_path: "{{tf_dir}}"
     state: "planned"
     plan_file: "plan.tfplan"
-    variables: "{{ kv_tf }}"
+    variables: "{{ kv_tf  }}"
```
Comment (jesse-dds) [inline ansible/roles/terraform/tasks/main.yml:25]: This looks like an accident

File: ansible/roles/terraform/tasks/main.yml [hunk 2]
```diff
@@ -10,27 +10,19 @@
...
 - name: sdhow tf_vars
   debug:
-    msg: "{{tfvars[deploy_tag]}}"
-
-
-- name: make k/v
-  set_fact:
-    kv_tf: "{{ kv_tf|default({}) | combine({ item.name : (item.value | regex_replace(\"'\",'\"'))  })   }}"
-  with_items: "{{tfvars[deploy_tag]}}"
+    msg: "{{kv_tf | to_nice_json}}"
```
Comment (jesse-dds) [inline ansible/roles/terraform/tasks/main.yml:17]: Pedantic but should these be surrounded by spaces like `{{ something }}` or tight like `{{something}}` - the original line looked like this `"{{ secrets_env  | from_yaml }}"`

File: ansible/roles/azure_keyvault/tasks/format_secrets.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+- set_fact:
+    kv_tf: "{{ kv_tf|default({}) | combine({ item.name : (item_no_quotes | regex_replace(\"'\",'\"'))  })   }}"
+  vars:
+    item_no_quotes: "{{ (item.value | regex_replace(\"'\",'')) if item.type=='' else (item.value|regex_replace(\"'\",'\"')) }}"
+  with_items: "{{unpacked}}"
+
+- name: show_kv_tf
+  debug:
+    msg: "{{ kv_tf  }}"
```
Comment (jesse-dds) [inline ansible/roles/azure_keyvault/tasks/format_secrets.yml:17]: Pedantic but deal, trailing spaces, and inconsistent spacing with surrounding braces `{{ ... }}` - I'm not sure what's proper here

File: ansible/roles/azure_keyvault/tasks/format_secrets.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+
+
+
+
```
Comment (jesse-dds) [inline ansible/roles/azure_keyvault/tasks/format_secrets.yml:4]: I think I noticed these leading newlines in other files too, how come we have these? Does this file get combined with other files into one mega-yaml?

**Comment by jesse-dds** [comment]
I pretty much only had formatting comments to contribute, will these all be fixed by https://github.com/dod-ccpo/atst/pull/1549/files?

**Comment by mikepw** [comment]
@jesse-dds respect your eye for detail here. formatted these spaces.  Looking for an Ansible formatter that will possibly address future slop-ness!

File: ansible/roles/azure_keyvault/tasks/format_secrets.yml [hunk 5]
```diff
@@ -0,0 +1,22 @@
+<<<<<<< HEAD
+
+
+- name: make k/v
+  set_fact:
+    kv_tf: "{{ kv_tf|default({}) | combine({ item : (secrets_env[item] | regex_replace(\"'\",'\"'))  })   }}"
+  with_items: "{{secrets_env}}"
+=======
```
Comment (jesse-dds) [inline ansible/roles/azure_keyvault/tasks/format_secrets.yml:8]: It looks like there's an unresolved merge collision in here

File: ansible/roles/azure_keyvault/tasks/main.yml [hunk 6]
```diff
@@ -16,4 +16,12 @@
 
 - name: get secrets
   import_tasks: get_secrets.yml
-  when: get_secrets == true
+<<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (jesse-dds) [inline ansible/roles/azure_keyvault/tasks/main.yml:20]: o lawd

**Pull request merged**